src/vs/editor/contrib/codeAction/browser/codeActionCommands.ts

```diff
@@ -536,7 +536,7 @@ registerEditorCommand(new CodeActionContribution({
 	kbOpts: {
 		weight: weight + 100000,
 		primary: KeyCode.UpArrow,
-		secondary: [KeyMod.CtrlCmd | KeyCode.UpArrow],
+		secondary: [KeyMod.CtrlCmd | KeyCode.KeyK],
```

Comment:
Does `ctrl+k` align with the keybindings for the suggest widget or other lists? I think that may just have been an example of a keybinding a user would like to add, not one we should support out of the box?